### PR TITLE
Ansible integration function. 

### PR DESCRIPTION
**Pull request by devsuds**
Ansible integration function.

### CODE REVIEW

File: fn_ansible/fn_ansible/util/config.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+   config_data = u"""[fn_ansible]
+[fn_ansible]
+playbook_dir=</full/path/to/your/playbook/directory>
+user_name=<USERNAME-OF-HOSTS>
+root_password=<PASSWORD-OF-HOSTS>
+hosts_path=</full/path/of/your/inventory/file>
+playbook_become_method=<SUPER-USER-METHOD e.g. sudo>
```
Comment (mscherfling) [inline fn_ansible/fn_ansible/util/config.py]: we may want to recommend that a company use our res-keyring capability to hide passwords:
root_password=^root_password.

see https://github.com/ibmresilient/resilient-reference/raw/master/developer_guides/Resilient%20IRP%20Function%20Developer%20Guide.pdf

File: fn_ansible/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  
+  
+  
+  
+
+## Using the Ansible Function
+
+Make sure that you have ansible v2.7 installed in your system(control machine). 
+One Function and one Workflow added as example we encourage you to explore that for better understanding.
+If you are using ansible-vault please provide full path of your password file( Recommended format below: ) -
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/README.md]: If not using vault then the field can be left blank

File: fn_ansible/fn_ansible/lib/ansible_api.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                        'become_user',
+                        'check',
+                        'listhosts',
+                        'listtasks',
+                        'listtags',
+                        'syntax',
+                        'sudo_user',
+                        'sudo',
+                        'diff'])
+    options = Options(connection=connection_type,
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/fn_ansible/lib/ansible_api.py]: Agreed.

File: fn_ansible/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  
+  
+  
+  
+
+## Using the Ansible Function
+
+Make sure that you have ansible v2.7 installed in your system(control machine). 
+One Function and one Workflow added as example we encourage you to explore that for better understanding.
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/README.md]: will remove it.

File: fn_ansible/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+  
+
+**You can use Ansible to automate three types of tasks:**
+
+  
+
+- Provisioning: Set up the various servers you need in your infrastructure.
+
+- **Configuration management: Change the configuration of an application, OS, or device; start and stop services; install or update applications; implement a security policy; or perform a wide variety of other configuration tasks.**
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/README.md]: I wanted to highlight where Ansible is more useful for information security

File: fn_ansible/tests/test_fn_ansible.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+from resilient_circuits import SubmitTestFunction, FunctionResult
+
+PACKAGE_NAME = "fn_ansible"
+FUNCTION_NAME = "fn_ansible"
+
+# Read the default configuration-data section from the package
+def get_mocked_config():
+  return """[fn_ansible]
+playbook_dir=/home/sudip/resilient-community-apps/fn_ansible/tests/test_utils
+user_name=sudip
```
[the remark this answers is not in the document]
Comment (devsuds) [reply fn_ansible/tests/test_fn_ansible.py]: I know i just missed that.

File: fn_ansible/fn_ansible/util/config.py (hunk 1 above)
Comment (devsuds) [reply fn_ansible/fn_ansible/util/config.py]: Done!

**Pull request closed without merging**